interpreter.go

```diff
@@ -11,13 +11,13 @@ import (
 	"fmt"
 )
 
-// TfLiteStatus represents TFLiteStatus
-type TfLiteStatus int
+// LiteStatus represents TFLiteStatus
+type LiteStatus int
 
-// options to represent TFLiteStatus
+// list of options to represent TFLiteStatus
```

Comment:
Is stricly necessary that this statuses are integers? It might be a good idea to make them strings.

Reply:
We can use a different definition, I used aligned idea from C library, we if we feel is more Go idiomatic & simple we can use a different approach,  is our envelope. 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/c/c_api_types.h

Reply:
I see, thanks for the reference, it makes sense to keep it as integers. If the main library changes (tensorflow C API), this sdk will change as well, reducing the amount of time developers need to refactor their code.